util/dvsim/SimCfg.py

```diff
@@ -298,10 +299,9 @@ def prune_items(items, marked_items):
         # Check if all items has been processed
         if items_list != []:
             log.error(
-                "The items %s added for run were not found in \n%s!\n \
-                Use the --list switch to see a list of available \
-                tests / regressions.", items_list, self.flow_cfg_file)
-            sys.exit(1)
+                "The items %s added for run were not found in \n%s!\n "
```

Comment:
These are very minor changes (dvsim errors) I had to deal with when running a custom "sw_access" regression target for all of top_earlgrey and IPs.